lib/spring/configuration.rb

```diff
@@ -49,4 +49,6 @@ def find_project_root(current_dir)
       end
     end
   end
+
+  self.verbose_output = true
```

Comment:
Calling the option "verbose" makes me think that it produces more than the normal amount of output. However the default is "verbose" mode.

I'd call this `Spring.quiet` instead, and have it default to false. Then you're opting in to quietness, rather than the opting out of verbosity. (I also think the `_output` bit is superfluous.)